index.js

```diff
@@ -176,18 +176,18 @@ function getExistingPath (command, opts) {
 
 module.exports._getNpmCache = getNpmCache
 function getNpmCache (opts) {
-  return which(opts.npm).then(npmPath => {
-    const args = ['config', 'get', 'cache', '--parseable']
+  return which('node').then(nodePath => {
+    const args = [opts.npm, 'config', 'get', 'cache', '--parseable']
```

Comment:
I think you can skip `which(node)` altogether and just use `process.argv[0]`, which will always point to a node executable?